docs/Development.md

```diff
@@ -140,39 +153,35 @@ The following is a recommended directory structure for a Pelorus exporter `<NAME
         └── test_<NAME>.py
 ```
 
-### Dev Environment Setup
-
-#### Python & Repo Setup
-
-After cloning the repo, you'll need a python version that's >= 3.9 but < 3.11.
-
-Running `make dev-env` should be enough to get you started.
+### Python development environment setup and and repo setup
+Install Python (version >= 3.9 but < 3.11) after cloning the repo. Running `make dev-env` should be enough to get started.
```

Comment:
Maybe make "Running make dev-env should be enough to get started" more visible? It's cool feature to install *all* required tools for development and testing on Linux or MacOS X systems.